libraries/testing/tester.cpp

```diff
@@ -375,7 +375,7 @@ namespace eosio::testing {
          // only block number is signaled, in forking tests will get the same block number more than once.
       });
       [[maybe_unused]] auto accepted_block_header_connection = control->accepted_block_header().connect([this](const block_signal_params& t) {
-            const auto& [block, id] = t;
+            [[maybe_unused]] const auto& [block, id] = t;
```

Comment:
Personally would like to see these compiler warnings not related to current changes come in a diff PR.

Reply:
Is next time good enough, or should I move them?

Reply:
Next time. Please mention in PR description.